### PR TITLE
Update type.md

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of the pull request

This pull request (PR) ...

## Affected pages

- https://experienceleague.adobe.com/en/docs/commerce-merchant-services/product-recommendations/admin/type

### CODE REVIEW

File: help/product-recommendations/type.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ These recommendation types are social-proof driven to help shoppers find what ot
...
+|Bought this, bought that|Recommends products that are infrequently purchased compared to the other products. This type will take into account the logical relations between products and will aim to recommend more interesting combinations versus top-purchased combinations. Displays highly relevant products shoppers can add to their cart by aggregating what other shoppers have bought with the current product.<br/><br/>**Where used:**<br/>- Product detail<br/>- Cart<br/>- Confirmation<br/><br/>**Suggested labels:**<br/>- Get everything that you need<br/>- Don't forget these<br/>- Frequently bought together|
```
Comment (krithikachandran) [inline help/product-recommendations/type.md:42]: I think "infrequently purchased" sends the wrong idea that we're ordering them to only return infrequently purchased SKUs. If the goal is to convey that this is not a simple co-occurence metric but a more sophisticated collaborative filtering algorithm, we should just say that. I suggest - `Recommends products that shoppers buy disproportionately more often with the currently viewed product. Note this is not a simple-occurrence metric but a more sophisticated collaborative-filtering machine-learning algorithm that looks for "interesting similarities" that are not skewed towards popular products. Displays highly relevant products shoppers can add to their cart by aggregating what other shoppers have bought with the current product.`

Comment (krithikachandran) [reply help/product-recommendations/type.md:42]: This should be added to the other collaborative filtering types as well and not just bought-bought

**Comment by erikmarr** [comment]
Migrated to internal repo. Thanks @shiftedreality and @krithikachandran!

**Comment by devops-devdocs** [comment]
Something went wrong

File: help/product-recommendations/type.md (hunk 1 above)
Comment (krithikachandran) [reply help/product-recommendations/type.md:42]: I know this PR is closed but I wanted to respond to Sandra's comment 
@sgonzalezmangana - I agree that the language might not be easy to follow. I am not sure how to provide an example without explaining LLR or probabilities which will only make this more complex. The simplest way we can differentiate is just specifying that this is based on shopper aggregate behavior across sessions and that if they are looking for in-session behavior then this rec type is not suitable for them.

**Comment by dshevtsov** [comment]
@devops-devdocs import the pr

**Comment by dshevtsov** [comment]
I'll temporarily reopen the PR for debugging the importing feature.

**Comment by dshevtsov** [comment]
@devops-devdocs import the pr

**Comment by devops-devdocs** [comment]
Something went wrong

**Comment by dshevtsov** [comment]
@devops-devdocs import the pr

**Comment by devops-devdocs** [comment]
Something went wrong

**Comment by dshevtsov** [comment]
@devops-devdocs import the pr

**Comment by devops-devdocs** [comment]
Something went wrong

**Comment by dshevtsov** [comment]
@devops-devdocs import the pr

**Comment by devops-devdocs** [comment]
Something went wrong

**Pull request closed without merging**